### DISCUSSION TITLE
У mintmanga сменился сайт

### DISCUSSION THREAD

**Discussion by HaruKunOFF** [lirix360/ReadmangaGrabber#87] [Q&A]
У mintmanga сменился сайт. С mintmanga.live на mintmanga.com. Из-за этого функционал перестал работать

**Comment by lirix360** [comment]
Домен добавлен в 2.1.9

**Comment by HaruKunOFF** [comment]
Отлично! А можно добавить поддержку https://z.allhen.online/ он тоже входит группу сайтов как readmanga и mintmanga?

**Comment by HaruKunOFF** [comment]
не выходит использовать mintmanga.com, так как нужны cookie но если авторизоваться на сайте, то сайт становиться m.mintmanga.live Получается его нужно добавить в программу

**Reply by lirix360** [reply]
Добавлен в 2.1.10

**Comment by HaruKunOFF** [comment]
Привет, просто я тебе наверно уже надоел, но теперь сайт https://23.mintmanga.live . Я скачать Source code и добавил в manga.go её вверху кода и внизу, но у меня не вышло. У меня либо на куки ругается, либо пишет ошибку "server returned 402 Payment Required"
Хотя там не о какой оплате речь не идет. Куки обновлял

**Reply by lirix360** [reply]
Версия 2.1.11.
В ней так же изменен механизм проверки cookies-файлов. Инструкция по новой механике сохранения файлов в Readme.
